package.json

```diff
@@ -137,7 +137,10 @@
 		"translate:dependencies": "rm -rf build/dependency-strings && mkdir -p build/strings && wp-babel-makepot './node_modules/@wordpress/**/build-module/**/*.js' --ignore '**/*.native.js' --base './' --dir './build/dependency-strings' --output './build/strings/calypso-dependencies.pot'",
 		"translate:calypso": "wp-babel-makepot './{client,packages,apps}/**/*.{js,jsx,ts,tsx}' --ignore '**/node_modules/**,**/test/**,**/*.d.ts' --base './' --dir './build/strings' --output './public/calypso-strings.pot'",
 		"tsc": "NODE_OPTIONS='--max-old-space-size=4096' tsc",
-		"typecheck": "yarn run tsc --project client",
+		"typecheck": "run-p 'typecheck-*'",
+		"typecheck-client": "yarn run tsc --project client",
+		"typecheck-apps": "find apps -name tsconfig.json | xargs -n 1 -P 4 -I {} yarn tsc --noEmit --project {}",
```

Comment:
This `find` command searches the entire `apps` subtree and that could backfire, especially if there is some nested `node_modules` folder in one of the sub-projects, with a NPM package that publishes a `tsconfig.json`.

`find -depth 2` will search only the first-level `apps` subfolder, equivalent to the `apps/*/tsconfig.json` glob used in the `unit-test-suite.mjs` CI script.

Reply:
That would be a good improvement. I'll open a pull request.

For what it's worth, this command isn't what's used in CI and would only be manually run on-demand on a developer's own computer, so not necessarily disruptive to shared builds. But still worth improving.

Reply:
Improved in #103507

Reply:
I agree it's not particularly important, but on the other hand it's just one extra CLI arg. Thanks for implementing the suggestion!